tests/test_filters.py

```diff
@@ -16,6 +16,13 @@
 RESOURCES = BASE_DIR / "tests" / "resources"
 
 
+# def test_save_medians(image_random, image_random_mask):
```

Comment:
Another mistake by me leaving this in, sorry.

Reply:
👍